src/backend_model.cc

```diff
@@ -539,6 +652,27 @@ TritonModel::SetConfiguredScheduler()
   return SetScheduler(std::move(scheduler));
 }
 
+Status
+TritonModel::UpdateConfiguredScheduler(
+    const std::vector<std::shared_ptr<TritonModelInstance>>& added_instances,
+    const std::vector<std::shared_ptr<TritonModelInstance>>& removed_instances)
+{
+  if (config_.has_sequence_batching()) {
+    SequenceBatchScheduler* sched =
+        dynamic_cast<SequenceBatchScheduler*>(scheduler_.get());
+    if (sched == nullptr) {
+      return Status(
+          Status::Code::INTERNAL,
+          "Unable to downcast from 'Scheduler' to 'SequenceBatchScheduler' "
+          "during scheduler update");
+    }
+    return sched->Update(added_instances, removed_instances);
+  }
+
+  // Non-sequence scheduler does not need to be updated.
```

Comment:
Add more description on why, something like instances are encapsulated in rate limiter and other scheduler doesn't need to be aware of the change as it only interact with the rate limiter.

Reply:
Yes. Updated: [Add more descriptions and improve function naming](https://github.com/triton-inference-server/core/pull/207/commits/686ebc95701d383515300cbf784fbba546cb9aef)